frontend/src/app/containers/ExperimentPage/DetailsDescription.js

```diff
@@ -0,0 +1,158 @@
+import parser from 'html-react-parser';
+import { Localized } from 'fluent-react/compat';
+import LocalizedHtml from '../../components/LocalizedHtml';
+
+import Warning from '../../components/Warning';
+import { experimentL10nId, formatDate } from '../../lib/utils';
+
+import GraduatedNotice from '../../components/GraduatedNotice';
+import IncompatibleAddons from './IncompatibleAddons';
+import { MeasurementsSection } from './DetailsOverview';
+
+import type {
+  DetailsDescriptionProps,
+  LocaleWarningType,
+  EolBlockProps
+} from './types';
+
+export default function DetailsDescription({
+  experiment,
+  graduated,
+  locale,
+  hasAddon,
+  installedAddons,
+  highlightMeasurementPanel
+}: DetailsDescriptionProps) {
+  const {
+    introduction,
+    warning,
+    details,
+    measurements,
+    video_url,
+    graduation_url,
+    completed
+  } = experiment;
+
+  const l10nId = pieces => experimentL10nId(experiment, pieces);
+
+  return (
+    <div className="details-description">
+      {completed && !graduated && <EolBlock {...{ experiment, l10nId }} />}
+      <IncompatibleAddons {...{ experiment, installedAddons }} />
+      <LocaleWarning {...{ experiment, locale, hasAddon }} />
+      {graduated && <GraduatedNotice {...{ graduated, graduation_url }} />}
+
+      {video_url &&
+        <iframe
+          width="100%"
+          height="360"
+          src={video_url}
+          frameBorder="0"
+          allowFullScreen
+          className="experiment-video"
+        />}
+      <div>
+        {!!introduction &&
+          <section className="introduction">
+            {!!warning &&
+              <div className="warning">
+                <Localized id={l10nId('warning')}>
+                  <strong>
+                    {warning}
+                  </strong>
+                </Localized>
+              </div>}
+            <LocalizedHtml id={l10nId('introduction')}>
+              <div>
+                {parser(introduction)}
+              </div>
+            </LocalizedHtml>
+          </section>}
+      </div>
+      <div className="details-list">
+        {details.map(({ image, copy, headline }, idx) =>
+          <div key={idx}>
+            <div className="details-image">
+              <img width="680" src={image} />
+              <p className="caption">
+                {headline &&
+                  <Localized id={l10nId(['details', idx, 'headline'])}>
+                    <strong>
+                      {headline}
+                    </strong>
+                  </Localized>}
+                {copy &&
+                  <Localized id={l10nId(['details', idx, 'copy'])}>
+                    <span>
+                      {parser(copy)}
+                    </span>
+                  </Localized>}
+              </p>
+            </div>
+          </div>
+        )}
+      </div>
+      {hasAddon &&
+        !graduated &&
+        measurements &&
+        <div>
+          <MeasurementsSection
```

Comment:
Second instance of `<MeasurementsSection>`, hidden except when in a narrow responsive view